src/media/js/previews-buttons.js

```diff
@@ -0,0 +1,59 @@
+/*
+    Arrow buttons for navigating back and forth in a preview tray that tell
+    Flipsnap what to do.
+
+    - Buttons disabled if there is no room to scroll left or right.
+    - We limit Flipsnap's max scroll position by 1 so we cant't scroll too far.
+    - If no room to scroll left or right (e.g., not enough previews), hide
+      buttons completely.
+*/
+define('previews-buttons', ['core/z'], function(z) {
+    function attach(slider, $container, opts) {
+        var isLightbox = opts && opts.isLightbox;
+
+        $container.find('.arrow-btn-prev, .arrow-btn-next').remove();
+
+        var $prevBtn = $('<a class="arrow-btn arrow-btn-prev"></a>');
+        var $nextBtn = $('<a class="arrow-btn arrow-btn-next"></a>');
+        var $previewBtns = $prevBtn.add($nextBtn);
+
+        function setPreviewBtnState() {
+            // Toggles whether buttons are hidden or disabled.
+            $prevBtn.add($nextBtn).addClass('arrow-btn-disabled');
+
+            var hasButton = false;
+            if (slider.hasNext() &&
+                slider.currentPoint !== slider._maxPoint - 1) {
+                // Only show the Next button if we can slide two more.
+                $nextBtn.removeClass('arrow-btn-disabled');
+                hasButton = true;
+            }
+            if (slider.hasPrev()) {
+                $prevBtn.removeClass('arrow-btn-disabled');
+                hasButton = true;
+            }
+
+            if (!isLightbox) {
```

Comment:
I should probably change the logic since all buttons start as visible, so I should ```hide()``` based off of hasButton rather than ```toggle```.

I'll add a comment, but I'm intending to not show the buttons at all (except on lightbox) if there's like only one or two previews.